### ISSUE TITLE
[BUG] bad contrast for active selection

### ISSUE THREAD

**Issue by mknudsen** [paperless-ngx/paperless-ngx#5886]
### Description

The contrast for active selection is very, very low and not easily recognisable. 

I think this violates the web content accessibility guidelines, see https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

See example, hard to see wether Lieferschein (existing document type) or the other line is selected.

![Screenshot 2024-02-24 at 17 52 03](https://github.com/paperless-ngx/paperless-ngx/assets/564654/af7b771e-8ef0-4b06-8c5f-6d5f2e172261)

### Steps to reproduce

1. edit document
2. enter letters of existing document type
3. try to select the thing you want

### Webserver logs

```bash
no relevant logs, this is a styling issue
```

### Paperless-ngx version

2.5.3

### Host OS

Linux knudstore 6.1.63-production+truenas #2 SMP PREEMPT_DYNAMIC Tue Jan 23 22:31:02 UTC 2024 x86_64 GNU/Linux

### Installation method

Docker - official image

### Browser

Chrome

### Other

using dark mode

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.
